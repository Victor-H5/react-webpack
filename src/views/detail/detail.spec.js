import React from 'react';
import { shallow } from 'enzyme';
import Detail from './detail';

describe('About component', () => {
	it('should match snapshot', () => {
		const wrapper = shallow(<Detail />);
		expect(wrapper).toMatchSnapshot();
	});

	it('should have Detail', () => {
		const wrapper = shallow(<Detail />);
		expect(wrapper.text()).toBe('Detail');
	});
});
