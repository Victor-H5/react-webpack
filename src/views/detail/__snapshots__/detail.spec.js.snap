// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should match snapshot 1`] = `
<div>
  Detail
</div>
`;
